HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb

```diff
@@ -0,0 +1,18 @@
+print 'Enter the value of the hours hand: '
+hours = gets.to_i
+print 'Enter the value of the minutes hand: '
+minutes = gets.to_i
+def angle_time(hours, minutes)
+  # Degrees the hour hand passes in a minute
+  hour_hand = 1 / (12 * 60.0) * 360
+  # Degrees the minute hand for a minute
+  minutes_hand = 360 * 1 / 60
+  # Clockwise angle
+  hour_angle = hour_hand * (minutes + hours * 60)
```

Comment:
You don't need this comment if you think that the name is not clear just rename the variable, for example:
```suggestion
  clockwise_angle= hour_hand * (minutes + hours * 60)
```

Reply:
Changed the name of variables